fuseops/ops.go

```diff
@@ -356,6 +356,7 @@ type CreateFileOp struct {
 	// later call to ReleaseFileHandle.
 	Handle    HandleID
 	OpContext OpContext
+	OpenFlags fusekernel.OpenFlags
```

Comment:
Can you add some documentation to this field? Does this field contain all flags passed as the second argument to the open() call, or just a subset?